### ISSUE TITLE
Removing client doesn't remove all assets that are connected to that client

### ISSUE THREAD

**Issue by sosojni** [sentora/sentora-core#68]
After removing test client, i can still find following folders and files on disk:

/etc/zpanel/panel/modules/webalizer_stats/stats/CLIENTNAME
/etc/zpanel/panel/modules/webalizer_stats/stats/CLIENTNAME/DOMAIN.com
/etc/zpanel/configs/bind/zones/DELETEDCLIENTSDOMAIN.com.txt
/var/zpanel/logs/domains/CLIENTNAME
/var/zpanel/logs/domains/CLIENTNAME/DOMAIN.com-access.log
/var/zpanel/logs/domains/CLIENTNAME/DOMAIN.com-error.log
/var/zpanel/logs/domains/CLIENTNAME/DOMAIN.com-bandwidth.log
/var/zpanel/logs/domains/CLIENTNAME/SUBDOMAIN.DOMAIN.com-access.log
/var/zpanel/logs/domains/CLIENTNAME/SUBDOMAIN.DOMAIN.com-error.log
/var/zpanel/logs/domains/CLIENTNAME/SUBDOMAIN.DOMAIN.com-bandwidth.log
/var/zpanel/vmail/DOMAIN.com

If client's web page was very active, you have bunch of logs / stats / mail on your disk that is taking up space you can use for something more useful. This gets troublesome on small VPS where you have active "traffic" of adding / removing clients. Suddenly you'r out of disk space and you have no idea why.

**Comment by yusufn** [comment]
Is there any "quick fix" for this? I need to re-register a customer account (which has been previously deleted and is not listed in clients) but when I try Sentora states the email address is in use by another account.
Thanks.

**Comment by bobsta63** [comment]
Yeah, the "quock fix" would be yo Open up the x_accounts table in the sentora_core database and delete the row from the table that has that email address... Then re-create the account and all should be fine :)

Cheers,
Bobby

**Comment by yusufn** [comment]
Wow. I popped to the shop just after posting this, just arrived home and valid response already. Confirmed "quick fix".
Thanks Bobby.

**Comment by bobsta63** [comment]
No problem buddy :)
